lib/ace/lib/event.js

```diff
@@ -115,6 +115,29 @@ exports.capture = function(el, eventHandler, releaseCaptureHandler) {
     return onMouseUp;
 };
 
+exports.addTouchMoveListener = function (el, callback) {
+    if ("ontouchmove" in el) {
+        var startx, starty;
+        exports.addListener(el, "touchstart", function (e) {
+            var touchObj = e.changedTouches[0];
+            startx = parseInt(touchObj.clientX);
+            starty = parseInt(touchObj.clientY);
+        });
+        exports.addListener(el, "touchmove", function (e) {
+            var factor = 1,
+            touchObj = e.changedTouches[0];
+
+            e.wheelX = -(parseInt(touchObj.clientX) - startx) / factor;
```

Comment:
I saw that pull request, but noticed it was removed, and was for an iOS application. So I have nothing to go off of there. 
In regards to touchObj.clientX not being a number, if that's ever the case, then the browser is not following W3 standards (http://www.w3.org/TR/touch-events/#idl-def-Touch).